tendermint/src/block/height.rs

```diff
@@ -7,25 +7,27 @@ use std::{
 
 /// Block height for a particular chain (i.e. number of blocks created since
 /// the chain began)
+/// 
+/// A height of 0 represents a chain which has not yet produced a block.
 #[derive(Copy, Clone, Eq, Hash, PartialEq, PartialOrd, Ord)]
 pub struct Height(u64);
 
 impl Height {
     /// Convert `u64` to block height.
-    ///
-    /// Note that 0 is not a valid block height.
+    /// 
+    /// Note that this method will never error and is just for backwards
+    /// compatibility.
     pub fn try_from_u64(n: u64) -> Result<Self, Error> {
```

Comment:
I don't think this method needs to exist anymore since the `From` impl always works. Or if you want to keep it around, it should probably just be `new` which returns `Self`

Reply:
✅